docs/versioned_docs/version-3.5/30-administration/10-configuration/40-autoscaler.md

```diff
@@ -27,7 +27,7 @@ services:
       - WOODPECKER_MIN_AGENTS=0
       - WOODPECKER_MAX_AGENTS=3
       - WOODPECKER_WORKFLOWS_PER_AGENT=2 # the number of workflows each agent can run at the same time
-      - WOODPECKER_GRPC_ADDR=https://grpc.your-woodpecker-server.tld # the grpc address of your woodpecker server, publicly accessible from the agents
+      - WOODPECKER_GRPC_ADDR=grpc.your-woodpecker-server.tld # the grpc address of your woodpecker server, publicly accessible from the agents. See https://woodpecker-ci.org/docs/administration/configuration/server#caddy for an example of how to expose. Do not include "https://" in the value.
```

Comment:
```suggestion
      - WOODPECKER_GRPC_ADDR=grpc.your-woodpecker-server.tld # the grpc address of your woodpecker server, publicly accessible from the agents. See https://woodpecker-ci.org/docs/administration/configuration/server#caddy for an example of how to expose. Do not include `https://` in the value.
```

Reply:
This is a comment in a yaml codeblock. Using backticks here doesn't make sense IMO